onmt/transforms/transform.py

```diff
@@ -224,9 +224,9 @@ def batch_apply(self, batch, is_train=False, **kwargs):
                 break
         return batch
 
-    def apply_reverse(self, translated):
+    def apply_reverse(self, translated, side="tgt"):
```

Comment:
I am not sure this is correct.
IIRC apply_reverse was historically used only in translator, and the passed arg "translated" has no 'side' I think, it is only the translated prediction. maybe needs to be checked.

Reply:
It's not useful, I wanted to detokenize the source but we don't need to do that.